website/pages/docs/plugins/sources/terraform/_meta.json

```
@@ -0,0 +1,8 @@
+{
+  "overview": "Overview",
+  "tables": {
+    "title": "Tables",
```

```suggestion
    "title": "Tables ↗",
```

I think it would good to indicate on all the Tables links that the links open in a new window by adding a little unicode arrow.

I'd like to bring the list of tables onto the site as well (not necessarily the columns, though), but we can do that in a follow-up.